client/src/components/ItemDetail/ItemDetail.tsx

```diff
@@ -81,34 +119,71 @@ function ItemDetail({
       >
         <CloseIcon />
       </IconButton>
-      <div style={{ width: "100vw", height: "60vh", overflow: "hidden" }}>
-        <img
-          src={item.media ? item.media : PLACEHOLDER_ITEM_MEDIA}
-          style={{ height: "100%", width: "100%", objectFit: "cover" }}
-        />
-      </div>
+      <div
+        style={{
+          width: "100vw",
+          height: `${Math.floor((1 - footerPercentageHeight) * 100)}vh`,
+          overflow: "hidden",
+          backgroundPosition: "center",
+          backgroundSize: "cover",
+          backgroundImage: `url('${
+            item.media ? item.media : PLACEHOLDER_ITEM_MEDIA
+          }')`,
+        }}
+      />
       <div style={{ width: "100vw" }}>
         {/*TODO(#198): zIndex layering helper class needed to manage layers on page*/}
         <div
           style={{
             position: "absolute",
             zIndex: 1003,
             width: "100%",
-            height: `${cornerRoundingPercentage}vw`,
-            marginTop: `-${cornerRoundingPercentage}vw`,
-            borderTopLeftRadius: `${cornerRoundingPercentage}vw`,
-            borderTopRightRadius: `${cornerRoundingPercentage}vw`,
+            height: `${cornerRoundingRadius}`,
+            marginTop: `-${cornerRoundingRadius}`,
+            borderTopLeftRadius: `${cornerRoundingRadius}`,
+            borderTopRightRadius: `${cornerRoundingRadius}`,
             backgroundColor: "#FFFFFF",
+            textAlign: "center",
           }}
-        />
+        >
+          {/*Floating Expand button*/}
+          {isOverflow && (
+            <Button
+              fullWidth={true}
+              style={{ height: `${cornerRoundingRadius}` }}
```

Comment:
nit: remove constant and replace with 24px. We dont want to make constants for css

Reply:
is there a reason for this? Wouldn't it be better if we could configure this css attribute perhaps from a field in [`themeConfig`](https://github.com/googleinterns/scan-and-go/blob/98cd5f7287339a01c2ea83519695b0524d8d0eaa/client/src/theme.ts#L65)

Reply:
for something that is used in multiple places like a theme - the constant is fine, but since this is restricted to only this file i suggested to inline it! If you think this is repeated used, adding it to a style config file, then importing and using is fine as well!

Reply:
I see, yeah this describes the extent of rectangular div cornering which we might want to keep consistent if we have to use elsewhere. I will move it into a field part of the `themeConfig` object.